### ISSUE TITLE
Please provide option to add PNG image as signature in the signature widget

### ISSUE THREAD

**Issue by ghoshpushpendu** [MatthiasValvekens/pyHanko#64]
Steps I Followed - 

add Signature Placeholder 
`pyhanko sign addfields --field  PAGE/X1,Y1,X2,Y2/NAME input.pdf output.pdf`

Sign PDF
`pyhanko sign addsig --field Sig1 pemder \
    --key key.pem --cert cert.pem input.pdf output.pdf`

This signs the PDF and create a rectangle with the signature.

But is there a way to add signature image like png there ?

**Comment by ghoshpushpendu** [comment]
I am new to python and a node js developer - so I think it ll take time for me to understand your code. @MatthiasValvekens

**Comment by MatthiasValvekens** [comment]
Yes, that's possible; see here: https://pyhanko.readthedocs.io/en/latest/cli-guide/config.html#styles-for-stamping-and-signature-appearances and https://pyhanko.readthedocs.io/en/latest/api-docs/pyhanko.stamp.html#pyhanko.stamp.BaseStampStyle.

In particular, note the following:

 - The `background` entry in a style definition can take a PNG image as its argument (or even another PDF).
 - You can leave the `stamp-text` empty.
 - There's a `background-opacity` option (which is only documented in the [API reference](https://pyhanko.readthedocs.io/en/latest/api-docs/pyhanko.stamp.html#pyhanko.stamp.BaseStampStyle) at this point, but it works in config as well) that allows you to have the background displayed with full opacity.

Combining these, it's possible to set up a signature appearance with only an image and no text:

```yaml
stamp-styles:
    bitmap-notext:
        type: text
        stamp-text: ""
        background: image.png
        background-opacity: 1
```

Would that work for you?
